types/src/on_chain_config/consensus_config.rs

```diff
@@ -259,16 +289,18 @@ impl OnChainConsensusConfig {
     pub fn enable_validator_txns(&mut self) {
         let new_self = match std::mem::take(self) {
             OnChainConsensusConfig::V1(config) => OnChainConsensusConfig::V3 {
-                alg: Jolteon {
+                alg: ConsensusAlgorithmConfig::JolteonV2 {
                     main: config,
                     quorum_store_enabled: false,
+                    order_vote_enabled: false,
```

Comment:
let's enable order vote by default in smoke tests?

Reply:
As we need to disable order votes by default in the release, I set the flag to false here.
I created another PR (https://github.com/aptos-labs/aptos-core/pull/13166) stacked on top of this to enable the flag and test.